### ISSUE TITLE
Private credentials were accidentally made public

### ISSUE THREAD

**Issue by almereyda** [allmende/ofn-docker#2]
This repository contains accidentally committed database credentials, which need to be factored out.

**Comment by elf-pavlik** [comment]
also https://github.com/allmende/ofn-docker/blob/master/docker-compose.yml#L41

😉

**Comment by almereyda** [comment]
Yes, and https://github.com/allmende/ofn-docker/blob/master/application.yml#L5
